tern/analyze/docker/dockerfile.py

```diff
@@ -333,3 +335,70 @@ def write_locked_dockerfile(dfile, destination=None):
         file_name = constants.locked_dockerfile
     with open(file_name, 'w') as f:
         f.write(dfile)
+
+
+def check_mutistage_dockerfile(dfobj):
+    """Given a dockerfile object, return if it is a multistage dockerfile."""
+    from_line = [1 for c in dfobj.structure if c['instruction'] == 'FROM']
+    return len(from_line) > 1
+
+
+def split_multistage_dockerfile(dfobj):
```

Comment:
I am getting 3 Dockerfile objects for a 2 stage dockerfile: https://github.com/anuvu/zot/blob/master/Dockerfile

Reply:
In the new commit, i have removed this function. We are now spliting the dockerfile for building, and `split_multistage_dockerfile()` gets the dfobj for analyze, so i will fix this function while i am working on the analyze.